mitodl/open-discussions
# Normalize course subjects to OCW topics/subtopics

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes #2903

#### What's this PR do?

- Adds a crosswalk mapping for MITx -> OCW topics
- Removes crosswalk mapping for OCW -> MITx topics
- Updates profession education mapping
- Adds OCW subtopics to ingested course topics

#### How should this be manually tested?

- Verify the application still works against data ingested under the `master` branch
- Run:
  - `./manage.py backpopulate_ocw_data`
  -  `./manage.py backpopulate_edx_data`
  -  `./manage.py backpopulate_mitpe_data`
  -  `./manage.py backpopulate_xpro_data`
- After all the above have completed, run `./manage.py backpopulate_youtube_data`
- Verify courses and videos now have updated subjects and the search works as you'd expect
- Your subject facet should match the screenshot below

#### Screenshots (if appropriate)
<details><summary>Expanded Subjects Facet</summary>

![Screenshot_2020-05-11 MIT Open Learning](https://user-images.githubusercontent.com/28598/81600030-fdc2c900-9396-11ea-872b-c34344063337.png)
</details>

Note: I wasn't able to capture a mobile screenshot of the entirety of the list, so suffice it to say it's the current mobile UI, but the same long list as the desktop.
